src/calendar/day/basic/style.js

```diff
@@ -38,7 +53,7 @@ export default function styleConstructor(theme={}) {
       color: appStyle.selectedDayTextColor
     },
     disabledText: {
-      color: appStyle.textDisabledColor
+      opacity: .3
```

Comment:
We still want to use `appStyle.textDisabledColor` since people still using it when they override through the `theme` prop.